--- /usr/lib/python3.13/site-packages/liquidctl/driver/kraken3.py       2025-04-07 15:54:42.399225195 +0100
+++ kraken3.py  2025-04-07 15:53:59.676402140 +0100
@@ -594,6 +594,18 @@
                 "lcd_resolution": (240, 240),
             },
         ),
+        (
+            0x1E71,
+            0x3012,
+            "NZXT Kraken 2024 Elite RGB",
+            {
+                "speed_channels": _SPEED_CHANNELS_KRAKEN2023,
+                "color_channels": _COLOR_CHANNELS_KRAKEN2023,
+                "hwmon_ctrl_mapping": _HWMON_CTRL_MAPPING_KRAKENZ,
+                "bulk_buffer_size": 1024 * 1024 * 2,  # 2 MB
+                "lcd_resolution": (640, 640),
+            },
+        ),
     ]

     def __init__(
@@ -1154,3 +1166,4 @@
             ]
         )
         return response[14] == 0x1
+
